free-2.md

```diff
@@ -11,41 +11,81 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help
 them?_
 
+There are lots of instances where presentation slides are used as graphic design tools. This may be because Photoshop is expensive and hard to learn, and slides give enough drawing and diagram options for particular use cases. 
+
+However, some of these designs are intended for reuse and often have patterns in them. Like ContextFree, perhaps there is a way to declaratively make these slides/images without having to manually edit the information every time. The next section will give some more examples of use cases.
+
```

Comment:
ContextFree inspired me too. I think there are a lot of potential niches surrounding graphics and image manipulation